scripts/globals/mobskills/chemical_bomb.lua

```diff
@@ -33,4 +35,4 @@ function onMobWeaponSkill(target, mob, skill)
 
     -- This likely doesn't behave like retail.
     return typeEffectTwo
```

Comment:
I know you haven't touched this, but this comment might be addressed as we're about to scrutinize Proto-Ultima.